src/components/htmlComponent.ts

```diff
@@ -13,51 +17,42 @@ import * as RenderController from "../core/renderController";
 import * as Utils from "../utils";
 
 export class HTMLComponent extends AbstractComponent<HTMLElement> {
```

Comment:
Ideally, yea you'd just say I want to render this to this type of container, but Plottable's notion of anchoring and space measuring makes it necessary to divide, for now, the two concerns. If we get rid of anchoring and computeLayout then we can probably throw away a lot of this  code. I think chasing perfection in this corner of the code is going to be better solved by simplifying the anchor / setup behavior down the line. 


That said, there really isn't _that_ much duplication here, so i get that it isn't 100% perfect but it isn't too egregious. If we really want to throw away the duplicate code we need to revisit how anchoring is thought of